src/v3/src/transformer/oktaVerify/transformOktaVerifyChannelSelection.ts

```diff
@@ -84,6 +84,7 @@ export const transformOktaVerifyChannelSelection: IdxStepTransformer = ({
       options: {
         content: loc('oie.enroll.okta_verify.switch.channel.link.text', 'login'),
         contentClassname: 'switch-channel-link',
+        submitOnClick: true,
```

Comment:
I think you will also need to set this in these files: 

[transformOktaVerifyEnrollPoll](https://github.com/okta/okta-signin-widget/blob/0.1/src/v3/src/transformer/oktaVerify/transformOktaVerifyEnrollPoll.ts#L53)
[transformOktaVerifyEnrollChannel](https://github.com/okta/okta-signin-widget/blob/0.1/src/v3/src/transformer/oktaVerify/transformOktaVerifyEnrollChannel.ts#L89)
[ReminderPrompt.tsx](https://github.com/okta/okta-signin-widget/blob/0.1/src/v3/src/components/ReminderPrompt/ReminderPrompt.tsx#L95-L106)